### PR TITLE
Add license and update package.json

### PR DESCRIPTION
**Pull request by gideonthomas**
@Pomax r?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+    "url": "git+https://github.com/mozilla/thimble.mozilla.org.git"
+  },
+  "keywords": [
+    "mozilla",
+    "thimble",
+    "editor"
+  ],
+  "author": "Mozilla",
+  "license": "MPL-2.0",
+  "bugs": {
```
Comment (Pomax) [inline package.json]: I'd move this up to next to the repository section

File: package.json [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
+    "url": "https://github.com/mozilla/thimble.mozilla.org/issues"
+  },
+  "homepage": "https://github.com/mozilla/thimble.mozilla.org#readme",
```
Comment (Pomax) [inline package.json]: this should be the real homepage, not the code homepage, so https://thimble.mozilla.org

**Comment by gideonthomas** [comment]
@Pomax does this look good?

**Comment by Pomax** [comment]
looks good to me! r+

**Pull request merged**